server/src/user/dto/change-password-dto.ts

```diff
@@ -0,0 +1,21 @@
+import { IsNotEmpty, IsString, MaxLength, MinLength } from 'class-validator';
+
+export class ChangePasswordDto {
+  @IsString()
+  @IsNotEmpty()
+  @MinLength(8)
+  @MaxLength(50)
+  currentPassword: string;
+
+  @IsString()
+  @IsNotEmpty()
+  @MinLength(8)
+  @MaxLength(50)
+  newPassword: string;
+
+  @IsString()
+  @IsNotEmpty()
+  @MinLength(8)
+  @MaxLength(50)
+  confirmNewPassword: string;
```

Comment:
You don't need this!

Reply:
Fixed it.
716cdb3402c6a0a80e41db82e93ae2a64a3ba0f3